Add WebKit bindings

pojntfx opened the pull request:
This adds Soup, JSC and WebKit bindings by extracting the GIR files and adding a small fix to only add namespaces to non-primitive types. It works quite well; I've ported the Markdown renderer from Senbara over to it already: https://mastodon.social/@pojntfx/115705952141846665

jwijenbergh commented:
could you fix the unverified commits and add a note to the readme that this is now supported? Also nit: your second commit has two commas

pojntfx commented:
@jwijenbergh Thanks for the review! Fixed - the unverified commits was because my Git GUI didn't sign when cherry picking. I've added a list of the libraries we generate bindings for to the README :)
